app/services/exchange_rates/create_csv.rb

```diff
@@ -0,0 +1,71 @@
+module ExchangeRates
```

Comment:
An idiom in the app is to always name services with a Service suffix

Reply:
Yeah i mentioned this to Jeremy because seemed confusing to me? Is this enforced? Also noticed it with the `_factory` in factories? didnt seem much point as already prefixed under the factories folder.

Jeremy mentioned about `_service` in services that its to do with name clashes (potentially models) but then id see that as a code smell if something doesnt look like a service and naming is incorrect or approach? Happy to discuss this if my point doesnt make sense?

Reply:
Its mainly for consistency at this point. Not really any good reason. I'd be happy to have them all renamed to not include Service. Just being a mixed bag is odd

Reply:
Name clashes makes sense, yeah

Reply:
Yeah makes sense, i can change and if I really throw toys out pram and get round to it propose a folder change PR 😂

Reply:
:tada: